README.md

```diff
@@ -175,6 +185,68 @@ jobs:
           body: ${{ steps.calibre.outputs.markdown }}
 ```
 
+## Combined workflow
+
+You can combine all of the above into one all-encompassing workflow

```

Comment:
Whew. This is a hefty workflow example.

Would you expect people to copy and paste this example, or to edit it down? 

Could you explain how this should be used? Seeing as it's not a "copy-paste-see-ya-later" experience like the previous examples are I think it's important to add as much detail as we can here.

Reply:
Yeah took me a bit of time to get ti working, hence why I thought it would be useful for others. I changed that sentence to this:

> You can combine all of the above into one all-encompassing workflow to avoid having to set up separate workflows with a lot of duplication of configuration. This does require some GitHub Actions syntax workflow to ensure the right workflow runs at the right time, so this is shown below. The only piece that requires changing is the `example/example_repo` line which should be changed to your own repository.

Think that's enough explanation?

Reply:
Yep. That's great. Really helpful. 👍